lib/pymedphys/_trf/decode/config.json

```diff
@@ -17,6 +17,381 @@
+        "2489_220": "Y1 Leaf 30/Positional Error (mm)",
+        "2490_220": "Y1 Leaf 31/Positional Error (mm)",
+        "2491_220": "Y1 Leaf 32/Positional Error (mm)",
+        "2492_220": "Y1 Leaf 33/Positional Error (mm)",
+        "2493_220": "Y1 Leaf 34/Positional Error (mm)",
+        "2494_220": "Y1 Leaf 35/Positional Error (mm)",
+        "2495_220": "Y1 Leaf 36/Positional Error (mm)",
+        "2496_220": "Y1 Leaf 37/Positional Error (mm)",
+        "2497_220": "Y1 Leaf 38/Positional Error (mm)",
+        "2498_220": "Y1 Leaf 39/Positional Error (mm)",
+        "2499_220": "Y1 Leaf 40/Positional Error (mm)",
+        "2500_220": "Y1 Leaf 41/Positional Error (mm)",
+        "2501_220": "Y1 Leaf 42/Positional Error (mm)",
+        "2502_220": "Y1 Leaf 43/Positional Error (mm)",
+        "2503_220": "Y1 Leaf 44/Positional Error (mm)",
+        "2504_220": "Y1 Leaf 45/Positional Error (mm)",
+        "2505_220": "Y1 Leaf 46/Positional Error (mm)",
+        "2506_220": "Y1 Leaf 47/Positional Error (mm)",
+        "2507_220": "Y1 Leaf 48/Positional Error (mm)",
+        "2508_220": "Y1 Leaf 49/Positional Error (mm)",
+        "2509_220": "Y1 Leaf 50/Positional Error (mm)",
+        "2510_220": "Y1 Leaf 51/Positional Error (mm)",
+        "2511_220": "Y1 Leaf 52/Positional Error (mm)",
+        "2512_220": "Y1 Leaf 53/Positional Error (mm)",
+        "2513_220": "Y1 Leaf 54/Positional Error (mm)",
+        "2514_220": "Y1 Leaf 55/Positional Error (mm)",
+        "2515_220": "Y1 Leaf 56/Positional Error (mm)",
+        "2516_220": "Y1 Leaf 57/Positional Error (mm)",
+        "2517_220": "Y1 Leaf 58/Positional Error (mm)",
+        "2518_220": "Y1 Leaf 59/Positional Error (mm)",
+        "2519_220": "Y1 Leaf 60/Positional Error (mm)",
+        "2520_220": "Y1 Leaf 61/Positional Error (mm)",
+        "2521_220": "Y1 Leaf 62/Positional Error (mm)",
+        "2522_220": "Y1 Leaf 63/Positional Error (mm)",
+        "2523_220": "Y1 Leaf 64/Positional Error (mm)",
+        "2524_220": "Y1 Leaf 65/Positional Error (mm)",
+        "2525_220": "Y1 Leaf 66/Positional Error (mm)",
+        "2526_220": "Y1 Leaf 67/Positional Error (mm)",
+        "2527_220": "Y1 Leaf 68/Positional Error (mm)",
+        "2528_220": "Y1 Leaf 69/Positional Error (mm)",
+        "2529_220": "Y1 Leaf 70/Positional Error (mm)",
+        "2530_220": "Y1 Leaf 71/Positional Error (mm)",
+        "2531_220": "Y1 Leaf 72/Positional Error (mm)",
+        "2532_220": "Y1 Leaf 73/Positional Error (mm)",
+        "2533_220": "Y1 Leaf 74/Positional Error (mm)",
+        "2534_220": "Y1 Leaf 75/Positional Error (mm)",
+        "2535_220": "Y1 Leaf 76/Positional Error (mm)",
+        "2536_220": "Y1 Leaf 77/Positional Error (mm)",
+        "2537_220": "Y1 Leaf 78/Positional Error (mm)",
+        "2538_220": "Y1 Leaf 79/Positional Error (mm)",
+        "2539_220": "Y1 Leaf 80/Positional Error (mm)",
+        "2170_111": "Mlc Status/Actual Value (None)"
```

Comment:
did you want to put this as the column name after the "Y1 Leaf 80/Positional Error" column name entry (below)?

Reply:
Yes i believe this is used on the latest version of MR Linac (v4). But you can double check on the version you have files from. I believe this 2170_111 is present. I agree with the below suggestion with the dictionary filling with unknowns. And then look in the software for completion of config.json later.